lib/cocina/generator/schema_value.rb

```diff
@@ -16,7 +16,7 @@ def generate
       private
 
       def type
-        "Types::#{dry_datatype(schema_doc)}#{optional}#{default}#{enum}"
+        "#{dry_datatype(schema_doc)}#{optional}#{default}#{enum}"
```

Comment:
As above, the `Types::` prefixes have moved into `SchemaBase`.

Reply:
👌🏻